src/Detached.Mappers.EntityFramework/EFMapContext.cs

```diff
@@ -82,6 +89,16 @@ public EFMapContext(DbContext dbContext, QueryProvider queryProvider, MapParamet
                 {
                     switch (actionType)
                     {
+                        case MapperActionType.Update:
+
+                            if (ContextModel.ConcurrencyTokens.TryGetValue(entry.Metadata.Name, out string concurrencyTokenName))
```

Comment:
this is the fix for concurrency..
the mapper loads the entity just before writing the values, so it thinks everything is right and just replace the token value with the given one
with this patch, it fails as expected